lib/alchemy/resource.rb

```diff
@@ -84,6 +96,7 @@ def initialize(controller_path, module_definition=nil, custom_model=nil)
       @module_definition = module_definition
       @model = (custom_model or guess_model_from_controller_path)
       self.skipped_attributes = model.respond_to?(:skipped_alchemy_resource_attributes) ? model.skipped_alchemy_resource_attributes : DEFAULT_SKIPPED_ATTRIBUTES
+      self.restricted_attributes = model.respond_to?(:restricted_alchemy_resource_attributes) ? model.restricted_alchemy_resource_attributes : DEFAULT_RESTRICTED_ATTRIBUTES
```

Comment:
Just replace the constant with `[]` here and you don't need the constant any more.